README.md

```diff
@@ -1 +1,342 @@
+- [openssl](https://www.openssl.org/)
+- [helm](https://helm.sh/)
+
+Configure `gcloud` as a Docker credential helper:
+
+```shell
+gcloud auth configure-docker
+```
+
+##### Create a Google Kubernetes Engine (GKE) cluster
+
+Create a new cluster from the command line:
+
+```shell
+export CLUSTER=nuxeo-cluster
+export ZONE=us-east1-a
+
+gcloud container clusters create "$CLUSTER" --zone "$ZONE"
+```
+
+Configure `kubectl` to connect to the new cluster:
+
+```shell
+gcloud container clusters get-credentials "$CLUSTER" --zone "$ZONE"
+```
+
+##### Clone this repository
+
+Clone this repository:
+
+```shell
+git clone https://github.com/nuxeo/nuxeo-gcp-marketplace.git
+```
+
+##### Install the Application resource definition
+
+An Application resource is a collection of individual Kubernetes components,
+such as Services, Deployments, and so on, that you can manage as a group.
+
+To set up your cluster to understand Application resources, run the following
+command:
+
+```shell
+kubectl apply -f "https://raw.githubusercontent.com/GoogleCloudPlatform/marketplace-k8s-app-tools/master/crd/app-crd.yaml"
+```
+
+You need to run this command once.
+
+The Application resource is defined by the
+[Kubernetes SIG-apps](https://github.com/kubernetes/community/tree/master/sig-apps)
+community. The source code can be found on
+[github.com/kubernetes-sigs/application](https://github.com/kubernetes-sigs/application).
+
+#### Install the application
+
+Navigate to the `nuxeo-mp` directory:
+
+```shell
+cd deployer/chart/nuxeo-mp
+```
+
+##### Configure the app with environment variables
+
+Choose the namespace and instance name for the application.
+
+```shell
+export NAMESPACE=mynamespace
+export APP_INSTANCE=nuxeo
+```
+
+Configure the container image registry:
+
+```shell
+export REGISTRY=marketplace.gcr.io/hyland/nuxeo
+```
+
+Set up the image tag:
+
+It is advised to use stable image reference which you can find on
+[Marketplace Artifact Registry](https://marketplace.gcr.io/hyland/nuxeo).
+Example:
+
+```shell
+export TAG="1.0"
+```
+
+If needed, expose the Service externally and configure Ingress:
+
+```shell
+export PUBLIC_SERVICE_AND_INGRESS_ENABLED=true
+```
+
+> [!WARNING] Warning about GKE Ingress
+
+When using the built-in Ingress controller provided by Google Kubernetes Engine, [GKE Ingress](https://cloud.google.com/kubernetes-engine/docs/concepts/ingress), the `X-Forwarded-Port` request header is not set. Thus, you need to configure it as a [custom request header](https://cloud.google.com/load-balancing/docs/https/custom-headers#working-with-request).
+
+Alternately, you can:
+- Use another Ingress controller such as [NGINX Ingress Controller](https://docs.nginx.com/nginx-ingress-controller/).
+- Configure  [Cloud DNS for GKE](https://cloud.google.com/kubernetes-engine/docs/how-to/cloud-dns), in which case you need to set the `virtualHost` parameter of the `nuxeo` subchart with the DNS hostname, see [Install the Helm chart](#install-the-helm-chart).
```

Comment:
For information:
- https://hyland.atlassian.net/browse/NXP-32992
- https://github.com/nuxeo/nuxeo-lts/pull/2632 (I'm doing a bit of rework)

Reply:
We can always remove this warning once the improvement is merged!